namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-stg/resources/variables.tf

```diff
@@ -0,0 +1,37 @@
+variable "namespace" {
```

Comment:
this seems to be a duplicate of : 
namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-staging/resources/variables.tf

Reply:
stg has been removed now